cluster/saltbase/salt/kube-addons/kube-addon-update.sh

```diff
@@ -154,7 +154,11 @@ function wait-for-jobs() {
     local rv=0
     local pid
     for pid in $(jobs -p); do
-        wait ${pid} || (rv=1; log ERR "error in pid ${pid}")
```

Comment:
This turned out to be wrong because () opens a subshell and rv was lost. Now error is logged at the end.